backend/src/main/resources/application.yaml

```diff
@@ -25,7 +25,7 @@ graphql:
     scalars:
       Date:
         scalar-name: DateTime
-        format: yyyy-MM-dd'T'HH:MM:ss'Z'
+        format: yyyy-MM-dd'T'HH:MM:ss.SSS'Z'
```

Comment:
Fine by me but we're not actually using this right now—need to make some fields in `schema.graphqls` DateTime instead of String.